### ISSUE TITLE
Pilot new standards with 3 VFS teams

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#63507]

**Comment by erinrwhite** [comment]
[Suuuper rough draft](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2832498689/2023-11+New+experience+standards+VFS+teams+Research+plan) ready for your initial review @allison0034. Questions: is the approach okay? What can be clearer?

**Comment by allison0034** [comment]
Same for this, I think when I see the questions it will become clear if this will work or not.

**Comment by erinrwhite** [comment]
@allison0034 @rferguson13 [Survey questions added for VFS team survey](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2832498689/2023-11+New+experience+standards+VFS+teams+Research+plan#Method) as well. What's missing or unclear?

**Comment by rferguson13** [comment]
@erinrwhite Review complete! Just one comment.

**Comment by allison0034** [comment]
Looks great, I made a few small comments, but I'd really like to know if VFS teams have ever looked at the standards before a review or if they even care about them and why or why not.

**Comment by shiragoodman** [comment]
@erinrwhite I added some comments today (sorry, kept getting side-tracked). Overall, I think the plan looks good but I think we want to refine our questions for VFS teams. I'm not so certain that we want, or need, to test the old process and the new process simultaneously. I'd love to get other's opinions on this. My concerns are that it would take a significant amount of time, especially if we're aiming for 5 each (I looked on the calendar at our last 10 SRs... the earliest one was Sept 21. We'd also have to consider that not all VFS teams may opt in to the pilot). Also I have concerns that the teams participating in the new process vs. teams partaking in the old process may not be easy to compare... We'd have to consider how many times a specific person or individual has been through the SR, the maturity of their product, if they participated in DI or MPR, their team staffing, and even the number of findings we documented. Those things will all impact a person's opinions and I don't think our research plan accounts for that at all. 

I did propose in 1 of my comments instead that we only complete research on the new process, but we aim to complete research with at least 3 teams (or more) who have been through the old process. We can ask them specific questions comparing the old and new. This allows our testing window to likely be shorter, and ensures that we're not comparing apples and oranges (AKA teams with abc characteristics in the new process against teams with xyz characteristics in the old process). 

I would be happy to discuss this more over Zoom. Let me know!

**Comment by erinrwhite** [comment]
@shiragoodman thank you! This helps clarify a lot.

I agree, we should only complete research on the new process, but aim to complete research with at least 3 teams (or more) who have been through the old process. I can update the plan with this.

**Comment by erinrwhite** [comment]
I think this is ready to go for final review!

[2023-11: New experience standards (VFS teams): Research plan](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2832498689/2023-11+New+experience+standards+VFS+teams+Research+plan)

**Comment by rferguson13** [comment]
@erinrwhite Just one comment!

**Comment by shiragoodman** [comment]
@erinrwhite I added a few comments, most minor. My biggest question is around the agree/disagree scale. Can you poke around in Optimal Workshop to identify what our options are for the scale? I have concerns about providing "Neutral" as an option as I don't want VFS teams to just not respond. I tagged Allison bc I believe she is the one who had originally expressed these concerns in previous research. 

Other than that (which tbh, is also pretty minor) - this looks great! This should definitely be ready to share with Matt soon.

**Comment by erinrwhite** [comment]
@shiragoodman @allison0034 Nerd alert...

Here is the default suggested text for the Likert scale in Optimal. They allow you to choose either a 5-point or a 7-point scale:
<img width="306" alt="Screenshot 2023-11-20 at 10 49 31 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/755248/8dcf79e1-1141-4cc8-a5f3-c89fe170ae91">

Another option is just to not use this at all and make up our own scale with 4 options instead of 5.

I did a light google on this and apparently the [topic of whether to include a neutral option](https://www.reddit.com/r/statistics/comments/y64tj8/q_is_a_4point_likert_scale_scale_without_a/) is quite contentious among statisticians! Today I learned...

Bottom line, do we foresee people having neutral/no opinions about these questions? My thought is that we will. ("Dunno, don't care, just passing through.") And, that no-opinion data could also be useful for us to know. 

This would also allow us to use Optimal's Likert scale features out of the box, though it doesn't look like that feature does much more than make it one step easier to build out the survey. (No fancy data analysis capabilities, from what I can tell.)

**tl;dr** I "slightly agree" that including a neutral option is a good thing, but "strongly disagree" that it's a dealbreaker, and will go with what the team thinks!

**Comment by allison0034** [comment]
@erinrwhite I am fine with whatever you decide. If you find a "no opinion" answer is useful then use the neutral, if not, force them to choose. 7 point seems like a lot of feelings for this, I would suggest the 5 point.

**Comment by erinrwhite** [comment]
Was a bit late getting these to Matt this morning via Slack - tagging him in here @humancompanion-usds for his review when he has time. It's a holiday week so this'll likely move to the next sprint.

[Research plan](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2832498689/2023-11+New+experience+standards+VFS+teams+Research+plan)

**Comment by erinrwhite** [comment]
Closing - if we need to make changes based on OCTO feedback, we will address in a separate ticket.
